src/Surfnet/StepupSelfService/SamlStepupProviderBundle/Provider/ProviderRepository.php

```diff
@@ -59,4 +59,12 @@ public function get(string $providerName): Provider
 
         return $this->providers[$providerName];
     }
+
+    /**
+     * @return Provider[]
+     */
+    public function getAll(): array
```

Comment:
Awesome! 

Maybe nitpicky.. But for type specificity; I'd love to see you use a (to be created) collection object. I'd name that ProviderCollection.

Or that you more specifically define the array structure. Like such: `array<string, Provider>`